src/cryptography/x509/oid.py

```diff
@@ -61,7 +61,15 @@ def __hash__(self):
 
     @property
     def _name(self):
-        return _OID_NAMES.get(self, "Unknown OID")
+        return _OID_NAMES.get(self, ("Unknown OID", None))[0]
+
+    @property
+    def longname(self):
+        return _OID_NAMES.get(self, (None, None))[0]
+
+    @property
+    def shortname(self):
+        return _OID_NAMES.get(self, (None, None))[1]
```

Comment:
I think this reads a bit ugly, it's not immediately obvious to a casual reader what `[0]` and `[1]` are getting here. What about making all of these (shortname, longname) tuples be [`namedtuples`](https://docs.python.org/dev/library/collections.html#collections.namedtuple) instead? 

Namedtuples are lightweight, requiring no extra memory, and then you could have `.get(...).short` and `.get(...).long` or similar.